src/screens/Swap/FormOrHistory/History/index.js

```diff
@@ -122,6 +123,16 @@ const History = () => {
   return (
     <View style={[styles.root, { backgroundColor: colors.background }]}>
       <TrackScreen category="Swap" name="History" />
+      <View
+        style={[
+          styles.alertWrapper,
+          { backgroundColor: colors.white, borderColor: colors.lightFog },
```

Comment:
Why not include this in `styles.alertWrapper`?

Reply:
Because colors comes from a hook and changes per theme, it's not available on the styles as far as I know 🤔